nautobot/docs/models/dcim/deviceredundancygroups.md

```diff
@@ -0,0 +1,250 @@
+# Device Redundancy Groups
+Device Redundancy Groups are used to represent logical relationships between different devices. Typically, a redundancy group could represent a failover pair, failover group, or a load sharing cluster.
```

Comment:
I like the usage examples, but I would have expected a bit more information here about the model itself and how to set it up in the first, especially with regard to fields like `device_redundancy_group_priority` in the mix.